Dockerfile

```diff
@@ -15,7 +15,10 @@ VOLUME /etc/superset
 WORKDIR /home/superset
 
 # Install dependencies & create superset user
-RUN apt-get update && \
+RUN  echo "deb http://http.debian.net/debian jessie-backports main" | tee /etc/apt/sources.list.d/webupd8team-java.list && \
+    echo "deb-src http://http.debian.net/debian jessie-backports main" | tee -a /etc/apt/sources.list.d/webupd8team-java.list && \
```

Comment:
Why are these lines necessary?

Reply:
This installs Java 1.8 on the image which is required for PyAthena. It looks like there's another pull request that would achieve the same thing by using a different source image. If that one gets approved, you could use that instead and then PyAthena should just work by including it in apt-get making these lines unnecessary.